pkg/cmd/server/crypto/crypto.go

```diff
@@ -142,10 +142,8 @@ func DefaultCiphers() []uint16 {
 		// the next one is in the intermediate suite, but go1.8 http2isBadCipher() complains when it is included at the recommended index
 		// because it comes after ciphers forbidden by the http/2 spec
 		// tls.TLS_RSA_WITH_AES_128_CBC_SHA256,
-		tls.TLS_RSA_WITH_AES_128_CBC_SHA,        // forbidden by http/2
-		tls.TLS_RSA_WITH_AES_256_CBC_SHA,        // forbidden by http/2
-		tls.TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA, // forbidden by http/2
-		tls.TLS_RSA_WITH_3DES_EDE_CBC_SHA,       // forbidden by http/2
```

Comment:
I don't mind dropping these, but I'd like to leave commented out breadcrumbs explaining why we're not including things from the intermediate suite and why, e.g.
```
// tls.TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA, // forbidden by http/2, disabled to mitigate SWEET32 attack
// tls.TLS_RSA_WITH_3DES_EDE_CBC_SHA,       // forbidden by http/2, disabled to mitigate SWEET32 attack
```